dluhc-component-library/src/components/timetableForm/utils.ts

```diff
@@ -10,6 +17,41 @@ export const getTimetableDownload = (formData: FormState) => {
   )}`;
 };
 
+export const loadTimetable = async (url: string): Promise<FormState> => {
+  return fetch(url)
+    .then((res) => res.json())
+    .then((data) => formatTimetableData(data));
+};
+
+const formatTimetableData = (timetable: TimetableData): FormState => ({
+  [TITLE_KEY]: timetable.title,
```

Comment:
Do we need to do these first two or can we just spread timetable?

Reply:
Yeah exactly, i think most values should eventually be in there but for once like date edited that are generated rather than in the form we are fine for it to just be hidden since it may or may not exist